Engine/ac/screenoverlay.cpp

```diff
@@ -112,8 +112,8 @@ void ScreenOverlay::ReadFromSavegame(Stream *in, bool &has_bitmap, int32_t cmp_v
     else
     {
         in->ReadBool(); // [DEPRECATED] has alpha
-        if (!(in->ReadBool())) // screen relative position
-            _flags |= kOver_PositionAtRoomXY;
+        if (!(in->ReadBool())) // logically room relative == autoplace
```

Comment:
I think the comment here as it was made more sense because of the negation there? Also you are removing the `OVR_AUTOPLACE` and using autoposition as the new name and commenting autoplace. If I understood this is an old parameter that was removed in a new savegame update but the comment is being less helpful than the code itself.

Reply:
Ok, I changed to 
```
// "is screen relative pos" flag; historically room relative == autopos
```

Reply:
thanks! Reads easier now!